.travis.yml

```diff
@@ -13,6 +13,15 @@ matrix:
   fast_finish: true
 notifications:
   email: false
+jobs:
+  include:
+    - stage: "DocTest"
+      julia: 1.1
+      os: linux
+      script:
+        - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
+        - julia --project=docs/ --code-coverage=user docs/make.jl
+      name: "HTML"
```

Comment:
If you just remove this line this build stage will be called `DocTest` instead of `HTML`.

Reply:
Nice, I think it works now! I see. thanks for the help!